cirq/ops/named_qubit.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from cirq.ops import raw_types
+
+
+class NamedQubit(raw_types.QubitId):
+    """A qubit identified by name."""
+
```

Comment:
I'd document the ordering that this supports, since it is non-trivial.

Reply:
Done.